aws-dual-stack/README.md

```diff
@@ -0,0 +1,68 @@
+# How to create a new dual-stack cluster in aws
+
+### Prepare for installation
```

Comment:
Let's put something like "PRs welcome" as we have limited support for things like AWS and mac only.

Reply:
These are just samples scripts. DO NOT USE IN PROD!